### ISSUE TITLE
[BUG] Zutilo disabled after re-open the Zotero window

### ISSUE THREAD

**Issue by albert-ying** [wshanks/Zutilo#229]
## Replication steps:

1. Open Zotero
2. Use command+w to close the Zotero window (note that the Zotero is still working at the background)
3. Reopen the Zotero by clicking the App icon
4. The Zutilo shortcuts don't work in the re-opened window.

System: MacOS 12.4 M1

**Comment by qqobb** [comment]
This is an issue specific to Mac systems. It should be related to https://github.com/zotero/zotero/issues/1952 and https://github.com/jlegewie/zotfile/pull/595.

If you find a fix, please submit a PR.

**Comment by wshanks** [comment]
Yes, this is hard for me to test because I don't have easy access to MacOS. On Linux, when you close the window ctrl+W, the entire application exits (and strangely ctrl+Q doesn't work). Zutilo does try to set up a window listener to load future windows:

https://github.com/wshanks/Zutilo/blob/33723e3968b4809214b24d3608bbd1056e63010f/addon/chrome/content/zutilo/zutilo.js#L93-L106
